api/src/main/java/org/eclipse/microprofile/config/spi/Converter.java

```diff
@@ -67,7 +68,7 @@
     /**
      * Configure the string value to a specified type
      * @param value the string representation of a property value
-     * @return the converted value
+     * @return the converted value or null if value is null
```

Comment:
I understand the Converter handling null and perhaps converting it into a valud value. 
If that is the case then it needs documenting that the Converter needs to handle null input without throwing a NPE or any exception.

I can change my PR if you want?